examples/adder/tests/Makefile

```diff
@@ -33,10 +33,10 @@ PWD=$(shell pwd)
 
 ifeq ($(OS),Msys)
 WPWD=$(shell sh -c 'pwd -W')
-PYTHONPATH := $(WPWD)/../model;$(PYTHONPATH)
+PYTHONPATH := $(PWD)/../model;$(PYTHONPATH)
```

Comment:
You also need the change from `;` to `:` here (eg, pull `PYTHONPATH` out of the `if`)

Reply:
Actully I did look here: https://github.com/cocotb/cocotb/blob/master/examples/endian_swapper/tests/Makefile#L38